### DISCUSSION TITLE
Neighbourhood plans

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#37] [Planning consideration]
# Neighbourhood plans

We are starting to look at Neighbourhood plans to determine if there are any critical datasets we should prioritise that would either help with the plan-making or help people use the plans once they are made/adopted.

We think neighbourhood plans might fit within the [development plan](https://github.com/digital-land/data-standards-backlog/discussions/26) standard but we want to do some independent research first before making that call.

**Comment by colmjude** [comment]
## Examples of neighbourhood plans

* [St Albans have a set of "made" neighbourhood plans](https://www.stalbans.gov.uk/neighbourhood-planning)

**Comment by colmjude** [comment]
## Examples of the areas/boundaries neighbourhood plans cover

* [Wheathampstead Parish](https://www.stalbans.gov.uk/sites/default/files/documents/publications/planning-building-control/planning-policy/Wheathampstead_Parish.pdf) - used in the Wheathampstead neighbourhood plan

**Comment by colmjude** [comment]
## Process

Taken from the guidance:
Step 1: Designating neighbourhood area and if appropriate neighbourhood forum
Step 2: Preparing a draft neighbourhood plan or Order
Step 3: Pre-submission publicity and consultation
Step 4: Submission of a neighbourhood plan or Order proposal to the local planning authority
Step 5: Independent Examination
Steps 6 and 7: Referendum and bringing the neighbourhood plan or Order into force

### Details

**Designated Neighbourhood Areas**
This stage can take a while, as where no town or parish council is in place, a neighbourhood forum will need to be established, and they will need to apply to the LPA to be designated as a neighbourhood forum

**Draft Neighbourhood Plans**
Includes a period of pre-submission consultation (at least 6 weeks)

**Submitted Neighbourhood Plans**
This is submission to the LPA
The LPA publicise this for at least 6 weeks
The LPA then send this to the independent examiner (not PINS)

**Neighbourhood Plans at Examination Stage**
No public hearings (unless required), the examiner considers written reps
Examiner considers the 'referendum area' aka the plan boundary

**Neighbourhood Plans at Referendum stage**
The majority of votes are in favour of the neighbourhood plan, the LPA has 8 weeks to make the neighbourhood plan 'made' aka adopted

**Made (adopted) Neighbourhood Plans**
Becomes part of the authority's development plan

**Reply by stevenjmesser** [reply]
Ey up. Quick note to say that there's a very long list of neighbourhood plans available in the department. (I helped map it with the LGA in 2015/16.) The majority are designated forums and made plans, if memory serves.

**Comment by RichLimbrick** [comment]
Hi - I think the divergence and confusion over things like local plans and neighbourhood plans, and minerals plans and so on, is the terminology used. Central gov has a habit of using the terms "local plans" and "development plans" as synonyms. They are not. Plan documents, like a Local Plan or Neighbourhood Plan, is a "development plan document" or DPD. The "development plan" is simply made up of all the adopted "development plan documents" that apply to a given area. That area may be a site, or a region, or an entire local authority area, but a development plan area is defined by the query so you can't really predefine those areas in a standard. If you look at your development plan standard, and rename all the "development plan" labels as "development plan document" or DPD, then the standard can be used to record all types, including local plans (incoming regional plans?), the London Plan, Waste Plans, Neighbourhood plans, and so on. Those and the policies within them do have defined geometries. You can then query a site for example, and find the "development plan" that applies - in other words, all the DPDs that intersect it. You could have linked sets that deal with discreet types of DPD as you may want to hold different information about each type (referendums, reg 18s and so on).

**Comment by AngusMcNeillPeel** [comment]
Important to distinguish between Neighbourhood Areas - those which may or may not have prepared a Made Neighbourhood Plan. It is important for a Local Plan to take into consideration Made Neighbourhood Plans in its area as part of its preparation. This is relevant to the policies map discussions being had. Neighbourhood Plans can and often do contain spatial policies and allocations. Greater consistency on how Local Plans reflect Made Neighbourhood Plans in their policies map would be useful.
